docs/getting-started/cli.rst

```diff
@@ -93,6 +93,164 @@ The ``seals:`` prefix ensures that the seals package is imported and the environ
+
+    python -m imitation.scripts.train_rl with \
+            cartpole \
+            logging.log_dir=output/train_rl/CartPole-v1/expert \
+            total_timesteps=10000
+
+Now let's train a weaker agent. 
+
+.. code-block:: bash
+
+    python -m imitation.scripts.train_rl with \
+        cartpole \
+        logging.log_dir=output/train_rl/CartPole-v1/non_expert \
+        total_timesteps=1000     # simply training less
+
+
+We can evaluate each policy using the ``eval_policy`` script.
+For the expert: 
+
+.. code-block:: bash
+
+    python -m imitation.scripts.eval_policy with \
+            expert.policy_type=ppo \
+            expert.loader_kwargs.path=output/train_rl/CartPole-v1/expert/policies/final/model.zip \
+            environment.gym_id='CartPole-v1' \
+            environment.num_vec=1 \
+            logging.log_dir=output/eval_policy/CartPole-v1/expert
+
+which will return something like 
+
+.. code-block:: bash
+
+    INFO - eval_policy - Result: {
+            'n_traj': 74, 
+            'monitor_return_len': 74, 
+            'return_min': 26.0, 
+            'return_mean': 154.21621621621622, 
+            'return_std': 79.94377589657559, 
+            'return_max': 500.0, 
+            'len_min': 26, 
+            'len_mean': 154.21621621621622, 
+            'len_std': 79.94377589657559, 
+            'len_max': 500, 
+            'monitor_return_min': 26.0, 
+            'monitor_return_mean': 154.21621621621622, 
+            'monitor_return_std': 79.94377589657559, 
+            'monitor_return_max': 500.0
+        }
+    INFO - eval_policy - Completed after 0:00:12
+
+
+For the non-expert:
+
+.. code-block:: bash
+
+    python -m imitation.scripts.eval_policy with \
+            expert.policy_type=ppo \
+            expert.loader_kwargs.path=output/train_rl/CartPole-v1/non_expert/policies/final/model.zip \
+            environment.gym_id='CartPole-v1' \
+            environment.num_vec=1 \
+            logging.log_dir=output/eval_policy/CartPole-v1/non_expert
+
+
+.. code-block:: bash
+
+    INFO - eval_policy - Result: {
+            'n_traj': 355, 
+            'monitor_return_len': 355, 
+            'return_min': 8.0, 
+            'return_mean': 28.92676056338028, 
+            'return_std': 15.686012049373561, 
+            'return_max': 104.0, 
+            'len_min': 8, 
+            'len_mean': 28.92676056338028, 
+            'len_std': 15.686012049373561, 
+            'len_max': 104, 
+            'monitor_return_min': 8.0, 
+            'monitor_return_mean': 28.92676056338028, 
+            'monitor_return_std': 15.686012049373561, 
+            'monitor_return_max': 104.0
+    }
+    INFO - eval_policy - Completed after 0:00:17
+
+This will save the monitor csvs (one for each vectorised env, controlled by environment.num_vec). 
+We can load these with ``pandas`` and use the ``imitation.test.reward_improvement``
+module to compare the performances of the two policies.
+
+.. code-block:: python
```

Comment:
Is this the way we want to recommend? What about if multiple environments (this will only read the first monitor one)? It already prints out monitor return mean and SD so I think we should be able to do a statistical test directly from that.

Reply:
Would be great if we could have a CLI for evaluating reward improvements as well (instead of falling back to Python). The CLI could take care of iterating over contained csv files if you specify a folder instead of a single file.

Reply:
ok, I checked with @ernestum who apparently has a tool for that already. Therefore, it does not make sense to implement sth independently right now. 
Instead I vote for just adapting the current code-block to consume all generated monitor CSVs and leaving a todo comment to fix this when a CLI tool for handling significance testing becomes available.

Reply:
(implemented in 7f39df3fb384fa542f9c1a7746372ab3ef281f99)

Reply:
I like it a lot better that we promote doing random permutation tests here. The code surfaces the weirdness in which we store the output of the experiments but that is nothing we can fix in this PR.
Maybe add a bit more context in the print statement like:
```python
if is_signifcant_reward_improvement(...):
    print("The expert improved over the non-expert with 95% probability")
else:
    print("No significant (p=0.05) reward improvement of expert over non-expert")

Reply:
thanks for the suggestions, I took it.